src/index.html

```diff
@@ -19,13 +19,15 @@
   <body>
     <input
       type="text"
-      data-qa="keypress"
+      class="search-field search--large"
+      data-qa="big"
```

Comment:
The `data-qa` attribute for the big search form is incorrect. It should be `data-qa="keypress"` as per the task requirements.